shared/js/site.js

```diff
@@ -52,10 +52,39 @@ class Score {
                 // remove period at end for lookup in pagesSeenOn
                 let tosdrData = tosdr[match[0]]
 
+                if (!tosdrData) { return }
+
+                let matchGood = (tosdrData.match && tosdrData.match.good) || []
+                let matchBad = (tosdrData.match && tosdrData.match.bad) || []
+
+                // tosdr message
+                // 1. If we have a defined tosdr class look up the message in constants
+                //    for the corresponding letter class
+                // 2. If there are both good and bad points -> 'mixed'
+                // 3. Else use the calculated tosdr score to determine the message
+                let message = constants.tosdrMessages.unknown
+                if (tosdrData.class) {
+                    message = constants.tosdrMessages[tosdrData.class]
+                } else if (matchGood.length && matchBad.length) {
+                    message = constants.tosdrMessages.mixed
+                } else {
+                    if (tosdrData.score < 0) {
+                        message = constants.tosdrMessages.good  
+                    } else if (tosdrData.score === 0 && (matchGood.length || matchBad.length)) {
+                        message = constants.tosdrMessages.mixed
+                    } else if (tosdrData.score > 0 ) {
+                        message = constants.tosdrMessages.bad
+                    }
+                }
```

Comment:
Hmm, this is getting me a `unknown privacy practices` for facebook.com even though there's two negative reasons shown. Let me investigate further.

Reply:
Fixed it, turns out the constant values for `tosdrMessages.good` and `tosdrMessages.bad` were missing.